browser-test/src/support/ti_dashboard.ts

```diff
@@ -0,0 +1,52 @@
+import {Page} from 'playwright'
+
+// This class is to test Civiform in the TI path
+// It impements functionality to create a new client,Test if the given client is present in the dashboard
+// It also have a ClientInformation Interface to easily store clients
+// It requires the tests to be logged as a TI
+export class TIDashboard {
+  public page!: Page
+
+  constructor(page: Page) {
+    this.page = page
+  }
+
+  async gotoTIDashboardPage(page: Page) {
+    await page.click('text="Trusted intermediary dashboard"')
+  }
+
+  async createClient(client: ClientInformation) {
+    await this.page.fill('label:has-text("Email Address")', client.emailAddress)
+    await this.page.fill('label:has-text("First Name")', client.firstName)
+    await this.page.fill('label:has-text("Middle Name")', client.middleName)
+    await this.page.fill('label:has-text("Last Name")', client.lastName)
+    await this.page.fill('label:has-text("Date Of Birth")', client.dobDate)
+    await this.page.click('text="Add"')
+  }
+
+  async expectDashboardContainClient(client: ClientInformation) {
+    expect(
```

Comment:
Can you please share the command to do this?

Reply:
Mind sharing the command.

Reply:
I'm not sure I follow regarding what command you're asking about. I'd imagine this could be converted to something that:
  1) Finds all of the applicant rows through a query selector.
  2) For each row, does a subquery for the relevant field inside (similar to what each one is doing now).
  3) Confirms that ALL of the entries match and returns true if so, false otherwise.

Still not blocking, but likely would improve our confidence that everything's working.